MOZ_STACKWALK_SUPPORTS_LINUX requires GLIBC extensions, Android or
libunwind. Disable it unconditionally now as otherwise x86 build fails.

--- firefox-45.0/mozglue/misc/StackWalk.cpp.orig
+++ firefox-45.0/mozglue/misc/StackWalk.cpp
@@ -42,10 +42,7 @@
   (defined(XP_DARWIN) && \
    (defined(__i386) || defined(__ppc__) || defined(HAVE__UNWIND_BACKTRACE)))
 
-#define MOZ_STACKWALK_SUPPORTS_LINUX \
-  (defined(linux) && \
-   ((defined(__GNUC__) && (defined(__i386) || defined(PPC))) || \
-    defined(HAVE__UNWIND_BACKTRACE)))
+#define MOZ_STACKWALK_SUPPORTS_LINUX 0
 
 #if __GLIBC__ > 2 || (__GLIBC__ == 2 && __GLIBC_MINOR__ >= 1)
 #define HAVE___LIBC_STACK_END 1
